### PR TITLE
Update autocompletions

### PR DESCRIPTION
**Pull request by BitForger**
I updated the auto completions to support CC 1.7

### CODE REVIEW

**Comment by BitForger** [comment]
Can this be merged?

File: ComputerCraft.tmLanguage [hunk 1]
```diff
@@ -4,7 +4,7 @@
 <plist version="1.0">
 <dict>
 	<key>comment</key>
-	<string>ComputerCraft Syntax: Version 1.2</string>
+	<string>ComputerCraft Syntax: Version 1.0.4</string>
```
Comment (benanders) [inline ComputerCraft.tmLanguage]: Why did you change the version number? Incrementing it to 1.3 would've been logical, but 1.0.4 has no relation to anything.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-ST2-ComputerCraft-Package
+ComputerCraft-Support
```
Comment (benanders) [inline README.md]: Renaming the package ComputerCraft Support is misleding. I agree about removing the "ST2" part, but "Support" is questionable. I would prefer "ComputerCraft Package" instead.

File: completions.sublime-completions [hunk 3]
```diff
@@ -23,8 +23,11 @@
 
 	// Modem API
```
Comment (benanders) [inline completions.sublime-completions:24]: Could you make the indentation here consistent please?

File: ComputerCraft.tmLanguage (hunk 1 above)
Comment (BitForger) [inline ComputerCraft.tmLanguage]: Sorry I was playing with how the package control worked. I'll fix it.

File: completions.sublime-completions (hunk 3 above)
Comment (BitForger) [inline completions.sublime-completions:24]: Will do.

File: README.md (hunk 2 above)
Comment (BitForger) [inline README.md]: Fixed :)

**Comment by BitForger** [comment]
Latest work

File: completions.sublime-completions [hunk 4]
```diff
@@ -22,9 +22,12 @@
...
-		{"trigger": "modem.open", "contents": "modem.open(${0:int n})" },
-		{"trigger": "modem.close", "contents": "modem.close()" },
-		{"trigger": "modem.transmit", "contents": "modem.transmit(${1:int sChan}, ${2:int rChan}, ${3:message}" },
+		{ "trigger": "modem.open", "contents": "modem.open(${0:int n})" },
+        { "trigger": "modem.isOpen", "contents": "modem.isOpen(${1:int chan})"},
+		{ "trigger": "modem.close", "contents": "modem.close(${1:int chan})" },
+        { "trigger": "modem.closeAll", "contents": "modem.closeAll()"},
+		{ "trigger": "modem.transmit", "contents": "modem.transmit(${1:int sChan}, ${2:int rChan}, ${3:String message})" },
+        { "trigger": "modem.isWireless", "contents", "modem.isWireless()"},
```
Comment (xAnavrins) [inline completions.sublime-completions:30]: Comma after "contents" here is causing an error.

File: completions.sublime-completions [hunk 5]
```diff
@@ -88,122 +91,168 @@
...
+        { "trigger": "peripheral.wrap", "contents": "peripheral.wrap(${1:String side})" },
         // CC 1.5+
         { "trigger": "peripheral.getNames", "contents": "peripheral.getNames()" },
+        // CC 1.6+
+        { "trigger": "peripheral.find", "contents": "peripheral.find(${1:String type} ${2:[, function fnFilter(${3:name}, ${4:object})]}"},
+
+    // Printer API
+        { "trigger": "printer.newPage", "contents": "printer.newPage()"},
+        { "trigger": "printer.endPage", "contents": "printer.endPage()"},
+        { "trigger": "printer.write", "contents": "printer.write(${1:String text})"}.,
```
Comment (xAnavrins) [inline completions.sublime-completions:208]: Period at the end of this line is causing an error.
